### PR TITLE
chore: normalize makefile

### PR DESCRIPTION
**Pull request by zhujian7**
**What this PR does / why we need it**:

- change REPOSITORIES to REPOSITORY
- limit CPU while building

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @supereagle @hyy0322 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -136,43 +148,33 @@ build-web-local:
...
 
 container-web: build-web
-	@for registry in $(REGISTRIES); do                                                 \
-	  imageName=$(IMAGE_PREFIX)web$(IMAGE_SUFFIX);                                     \
-	  docker build -t $${registry}/$${imageName}:$(VERSION)                            \
-	    -f $(BUILD_DIR)/web/Dockerfile.local .;                                        \
-	done
+	imageName=$(IMAGE_PREFIX)web$(IMAGE_SUFFIX);                                       \
+	docker build -t ${REGISTRY}/$${imageName}:$(VERSION)                               \
+	  -f $(BUILD_DIR)/web/Dockerfile.local .
```
Comment (supereagle) [inline Makefile]: Missing `\` at the end of line?

File: Makefile [hunk 2]
```diff
@@ -136,43 +148,33 @@ build-web-local:
...
 
 container-web-local: build-web-local
-	@for registry in $(REGISTRIES); do                                                 \
-	  imageName=$(IMAGE_PREFIX)web$(IMAGE_SUFFIX);                                     \
-	  docker build -t $${registry}/$${imageName}:$(VERSION)                            \
-	    -f $(BUILD_DIR)/web/Dockerfile.local .;                                        \
-	done
+	imageName=$(IMAGE_PREFIX)web$(IMAGE_SUFFIX);                                       \
+	docker build -t ${REGISTRY}/$${imageName}:$(VERSION)                               \
+	  -f $(BUILD_DIR)/web/Dockerfile.local .
```
Comment (supereagle) [inline Makefile]: Missing `\` at the end of line?

File: Makefile [hunk 3]
```diff
@@ -86,48 +100,46 @@ $(GOLANGCI_LINT):
...
-	          yarn build';                                                             \
-	done
+	docker run --rm                                                                    \
+	  -v $(PWD)/web/:/app                                                              \
+	  -w /app                                                                          \
+	  -e SHELLOPTS=$(SHELLOPTS)                                                        \
+	    $(BASE_REGISTRY)/node:10.16-stretch                                            \
+	      sh -c '                                                                      \
+	        yarn;                                                                      \
+	        yarn build'
```
Comment (supereagle) [inline Makefile]: Missing `\` at the end of line?

File: Makefile [hunk 4]
```diff
@@ -86,48 +100,46 @@ $(GOLANGCI_LINT):
...
+	  -e GOOS=linux                                                                    \
+	  -e GOARCH=amd64                                                                  \
+	  -e GOPATH=/go                                                                    \
+	  -e CGO_ENABLED=0                                                                 \
+	  -e SHELLOPTS=$(SHELLOPTS)                                                        \
+	  $(BASE_REGISTRY)/golang:1.12.12-stretch                                          \
+	    /bin/bash -c 'for target in $(TARGETS); do                                     \
+	      go build -i -v -o $(OUTPUT_DIR)/$${target} -p $(CPUS)                        \
+	        -ldflags "-s -w -X $(ROOT)/pkg/server/version.VERSION=$(VERSION)           \
+	          -X $(ROOT)/pkg/version.COMMIT=$(COMMIT)                                  \
```
Comment (supereagle) [inline Makefile]:
```suggestion
	          -X $(ROOT)/pkg/server/version.COMMIT=$(COMMIT)                      \
	          -X $(ROOT)/pkg//serverversion.COMMIT=$(COMMIT)                      \
```

Comment (zhujian7) [reply Makefile]: fixed

File: Makefile (hunk 3 above)
Comment (zhujian7) [reply Makefile]: It's not needed, same as the others.

**Pull request merged**